adapters/zio-http/src/main/scala/caliban/ZHttpAdapter.scala

```diff
@@ -17,8 +17,10 @@ object ZHttpAdapter {
 
   def makeHttpService[R, E](interpreter: HttpInterpreter[R, E])(implicit
     serverOptions: ZioHttpServerOptions[R] = ZioHttpServerOptions.default[R]
-  ): HttpApp[R, Throwable] =
-    ZioHttpInterpreter(serverOptions).toHttp(interpreter.serverEndpoints[R, ZioStreams](ZioStreams))
+  ): App[R] =
+    ZioHttpInterpreter(serverOptions)
+      .toHttp(interpreter.serverEndpoints[R, ZioStreams](ZioStreams))
+      .withDefaultErrorResponse
```

Comment:
The only thing I don't like about this is that it throws away all information regarding the error so the user can't choose to take a different action on it

Reply:
True, but this is a one-liner they can easily copy. Otherwise it forces them to add this handler themselves which is more annoying.

Reply:
Also it seems the tapir code already convert errors like Bad Request into a Response, so it will throw away only the unhandled errors which should be very rare (since Caliban also catches errors on its own).